src/scripts.js

```diff
@@ -1,51 +1,53 @@
-import './styles.css';
-import apiCalls from './apiCalls';
+import './styles.css'
+import apiCalls from './apiCalls'
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
 import Recipe from './classes/Recipe'
 import RecipeRepository from './classes/RecipeRepository'
-import recipeData from './data/recipes';
- 
+import recipeData from './data/recipes'
 
 // --------------------------------------------- QUERY SELECTORS
 // const modalBtn = document.querySelector('#modalBtn');
-const modal = document.querySelector('#modal');
-const close = document.querySelector('#close');
-const recipeSection = document.querySelector('#recipeSection');
-
-
-
+const modal = document.querySelector('#modal')
+const close = document.querySelector('#close')
+const recipeSection = document.querySelector('#recipeSection')
 
 // --------------------------------------------- EVENT LISTENERS
 // modalBtn.onclick = () => {modal.style.display = "block"};
-close.onclick = () => {modal.style.display = "none";}
-window.onclick = (event) => {if (event.target == modal) {modal.style.display = "none"}};
-window.addEventListener("load", updateAllRecipeDisplay);
+close.onclick = () => {
+  modal.style.display = 'none'
+}
+window.onclick = (event) => {
+  if (event.target == modal) {
+    modal.style.display = 'none'
+  }
+}
+window.addEventListener('load', updateAllRecipeDisplay)
 
-  //will need some event bubblin for the cards we create dynamically later
+//will need some event bubblin for the cards we create dynamically later
 
 // --------------------------------------------- GLOBAL VARIABLES
-let recipeRepo = new RecipeRepository(recipeData);
-
-
-
-// --------------------------------------------- FETCH 
-
-
+const recipeRepo = new RecipeRepository(recipeData)
 
+// --------------------------------------------- FETCH
 
 // --------------------------------------------- FUNCTIONS
 function updateAllRecipeDisplay() {
-  recipeRepo.newAllRecipes.forEach(recipe => {
-    let tagsHTML = recipe.tags
-      .map(tag => {
-      return `<p class="recipe-tag">${tag}</p>`
-    })
-      .join(' ');
-    
-    recipeSection.innerHTML += `<section class="recipe-card">
+  recipeRepo.newAllRecipes.forEach((recipe) => {
+    const tagsHTML = recipe.tags
+      .map((tag) => {
+        return `<p class="recipe-tag">${tag}</p>`
+      })
+      .join(' ')
+
+    const recipeCard = document.createElement('section')
+    recipeCard.classList.add('recipe-card')
+    recipeCard.dataset.recipeId = `${recipe.id}`
```

Comment:
set the data id attribute to the `recipe id` so now all cards will have an id of their recipe id 
![image](https://user-images.githubusercontent.com/108706408/196605932-3ee70909-cd3c-43ec-9a0c-e7f7d5f4eae8.png)